lib/flutter_radar.dart

```diff
@@ -31,6 +48,34 @@ class Radar {
     }
   }
 
+  static attachListeners() async {
+    try {
+      if (Platform.isAndroid) {
+        print("attaching listeners");
+        await _channel.invokeMethod('attachListeners', {
+          'callbackDispatcherHandle':
+              PluginUtilities.getCallbackHandle(callbackDispatcher)?.toRawHandle()
+        });
+      } else {
+        // ios ignore
```

Comment:
@gabrielalao We had to make changes to the Android handler (`RadarFlutterPlugin.java`), I would have expected us to need to make the same changes for iOS (`RadarFlutterPlugin.m`). Does iOS still work on device in the background, and overall?  Why doesn't the iOS plugin need the same background channel processing that Android does?

Reply:
Since the iOS is working well in the background, so no need to change the same change as the android
@tjulien

Reply:
I was trying to make the same as change on the ios. I thought we can publish new version and let the client use new version for their product and I will raise a new PR for this update.